src/msg/async/dpdk/TCP.h

```diff
@@ -383,7 +383,8 @@ class tcp {
       uint32_t key[16];
       isn_secret () {
         for (auto& k : key) {
-          k = ceph::util::generate_random_number<uint32_t>();
+          //k = ceph::util::generate_random_number<uint32_t>();
+          k = ceph::util::generate_random_number();
```

Comment:
@chardan do you remember which helper you intended to use?

Reply:
also comment to removed before push

Reply:
In this case, I called the uint32_t version because the array "key" is of type uint32_t.

Reply:
thanks! @liu-chunmei , so per @chardan 's reply, i guess what he suggests is:
```c++
k = ceph::util::generate_random_number<uint32_t>(0, std::numeric_limits<uint32_t>::max());
```

Reply:
Go with that unless there's any actual issue (which I don't expect there will be).

Reply:
there is a build error if not change it, report generate_random_number, is not defined. In random.h there is no template <IntegerT> with no parameters required.

Reply:
@tchaikov @chardan updated the patch.